apkid/rules/dex/obfuscators.yara

```diff
@@ -188,22 +188,73 @@ rule allatori_demo : obfuscator
     $s and is_dex
 }
 
-rule aamo : obfuscator
+rule aamo_str_enc_nop : obfuscator
 {
   meta:
-    description = "AAMO"
+    description = "AAMO (String decryption function + interleaved NOPs)"
```

Comment:
Since this is a relatively rare obfuscator, I think this should be one rule.

You could have `$opcodes_nops` and `$opcodes` (without nops) in the same rule. The condition could be something like: `(1 of $opcodes*) and all of ($a, $b)`

Reply:
:+1:  oh ok, yes that makes sense to me

Reply:
Combined sounds good to me @CalebFenton.